pkgs/applications/networking/instant-messengers/teamspeak/client.nix

```diff
@@ -99,15 +99,15 @@ stdenv.mkDerivation rec {
   dontStrip = true;
   dontPatchELF = true;
 
-  meta = {
+  meta = with stdenv.lib; {
     description = "The TeamSpeak voice communication tool";
     homepage = "https://teamspeak.com/";
     license = {
       fullName = "Teamspeak client license";
       url = "http://sales.teamspeakusa.com/licensing.php";
```

Comment:
For me that redirects to https://www.teamspeak.com/en/features/licensing/

Reply:
Yeah; perhaps more annoyingly it doesn't actually list the client license there. I'm trying to find a better source.

Reply:
Updated. Originally I'd planned to just take the license out of the archive and bundle it in the repo, but it turns out it's the same as the T&Cs on their website.